mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -33,7 +33,12 @@
 _TSDB_RATE = "1/s"
 _CONTAINER = "users"
 
-V3IO_MEPS_LIMIT = 200
+V3IO_FRAMESD_MEPS_LIMIT = (
+    200  # Maximum number of model endpoints per single request when using V3IO Frames
+)
+V3IO_CLIENT_MEPS_LIMIT = (
+    150  # Maximum number of model endpoints per single request when using V3IO Client
+)
```

Comment:
Suggest to rename to make it clear :
```suggestion
V3IO_GET_ITEMS_FILTER_MEP_KEYS_MAX_NUMBER = (
    150  # Maximum number of MEPs per request when used as filter-expression in v3io nosql table (filter expression max size is 8192 bytes) 
)
```